protocol/messenger.go

```diff
@@ -4692,54 +4720,86 @@ func (m *Messenger) MarkAllReadInCommunity(communityID string) ([]string, error)
 
 // MuteChat signals to the messenger that we don't want to be notified
 // on new messages from this chat
-func (m *Messenger) MuteChat(chatID string) error {
-	chat, ok := m.allChats.Load(chatID)
+func (m *Messenger) MuteChat(MuteParams *requests.MuteParams) (int64, error) {
```

Comment:
The convention is not to use `MuteParams` as a name, we use the name of the call for the most part,
so the type should be `*requests.MuteChat`